### ISSUE TITLE
Invalid native attribute names

### ISSUE THREAD

**Issue by yo3jones** [Polymer/tools#2747]
Getting this browser error at runtime:

```
Uncaught DOMException: Failed to execute 'setAttribute' on 'Element': 'aria-live\' is not a valid attribute name.
```

It looks like the escape of the `$` is not necessary, which leads to a trailing back slash.

Example can be seen in [iron-a11y-announcer](https://github.com/PolymerElements/iron-a11y-announcer/blob/__auto_generated_3.0_preview/iron-a11y-announcer.js#L13)

```html
<div aria-live\$="[[mode]]">[[_text]]</div>
```

will work if the `\` is removed:
```html
<div aria-live$="[[mode]]">[[_text]]</div>
```
